components/CallToActionSection.vue

```diff
@@ -16,11 +24,44 @@
               <span class="hglt">
                 {{
                   content.subHeading ||
+                  description ||
                   'With the most intuitive form creator out there'
                 }}
               </span>
             </h2>
             <div
+              v-if="buttons && buttons?.length > 0"
+              class="d-flex align-items-center justify-content-center flex-wrap gap-3 mt-5"
+            >
+              <a
+                :href="
+                  buttons[0]?.link ||
+                  'https://app.formester.com/users/sign_up'
+                "
+                class="btn button mx-2"
+                :class="{
+                  'cta-button': buttons[0].type === 'Primary',
+                  'cta-button__invert': buttons[0].type === 'Secondary',
```

Comment:
Why are we getting buttons with index?

This doesn't look good. Is there any other way to without getting button by index?

Reply:
In the cms the buttons component is an array. As in some places we need multiple buttons. But in some places we have only one.
So We can map the list of buttons and show it. But, the only problem here is that if in the cms we add two buttons instead of one, then the UI might break.

Reply:
Check if there are any way to not break the UI even if there are multiple buttons.

Reply:
Well for that we need to add stylings that will support multiple buttons. So, that will be out of the scope of the PR.